static/docs/commands-reference/get-url.md

```diff
@@ -0,0 +1,164 @@
+# get-url
+
+Download or copy file or directory from any supported URL (for example `s3://`,
+`ssh://`, and other protocols) or local directory to the local file system.
+
+> Unlike `dvc import-url`, this command does not track the downloaded data
+> file(s) (does not create a DVC-file).
+
+## Synopsis
+
+```usage
+usage: dvc get-url [-h] [-q | -v] url [out]
+
+positional arguments:
+  url            (See supported URLs in the description.)
+  out            Destination path to put data to.
+```
+
+## Description
+
+In some cases it's convenient to get a data file or directory from a remote
+location. The `dvc get-url` command helps the user do so. The `url` argument
+should provide the location of the data to be downloaded, while `out` can be
+used to specify the (path and) file name desired for the downloaded data file or
+directory.
+
+> Like `dvc init`, this is one of the few commands that doesn't require an
```

Comment:
it's still feels strange to reference `dvc init` here tbh. It's not also that important that there are a few commands. I would skip the first half of the sentence. And it feels that we can make it a regular paragraph, not a note. It's important information that it's a "global" command.

Reply:
Updated in 88e434e but I did leave it as a note because I think it's not crucial information you need to know. I think of notes as things that the reader may chose to skip reading. You don't really need to know this but it's a nice-to-know. But I can make it a normal paragraph if you insist 😄

Reply:
Yep, my point is that it's important to understand in case of `dvc get` or `dvc get-url` that they do not require repo. They are solving different use cases - all other commands are about organizing the ML/data project, these two are about getting you model, for example in the production environment.

Reply:
Fair enough 🙂  Updated in e609e87